examples/vl3_basic/helm/vl3/values.yaml

```diff
@@ -3,22 +3,23 @@
 # Declare variables to be passed into your templates.
 
 registry: docker.io
-org: tiswanso
+org: mistrate
 tag: unified_api_ipam
-pullPolicy: Always
+pullPolicy: IfNotPresent
 
 cnns:
   nsr:
+    name: vl3-nse
     port: 5005
-    # addr: 
-#  domain: nsr.foo.com
+    addr: vl3-nse.cnns-cisco.com
+    cd: vl3-nse-connectivity-domain
 
+ipam:
```

Comment:
Add the ipam under the `cnns` key at the same level as the `nsr` key.  Also, add a field for specifying the `ipamServer`